substrate/client/keystore/src/local.rs

```diff
@@ -136,6 +136,13 @@ impl LocalKeystore {
 }
 
 impl Keystore for LocalKeystore {
+	/// Insert a new secret key.
```

Comment:
Should be put above the method declaration in the trait.

Reply:
I placed it here as the comments references the keystore construction using a password) which is a concept not referenced anywhere in the `Keystore` trait. I thought that talking about passwords in the trait could be misleading.